cmd/kubectl-k8ssandra/register/command.go

```diff
@@ -0,0 +1,72 @@
+package register
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+var RegisterClusterCmd = &cobra.Command{
+	Use:   "register [flags]",
+	Short: "register a data plane into the control plane.",
+	Long:  `register creates a ServiceAccount on a source cluster, copies its credentials and then creates a secret containing them on the destination cluster. It then also creates a ClientConfig on the destination cluster to reference the secret.`,
+	Run:   entrypoint,
+}
+
+func Init(cmd *cobra.Command, streams genericclioptions.IOStreams) {
```

Comment:
issue: You should not name functions with "init" or "Init" as this is a reserved function name in the Go. This function does not follow the expected semantics of "func init()" which has a special handling in the Go.

Reply:
Ah, I'd forgotten. Thank you...